src/components/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -190,34 +189,19 @@ export default {
 			rows: null,
 			isVisitor: true,
 			loanRowsCount: 4,
-			loanUseMaxLength: 100,
-			anonymizationLevel: 'none',
-			shareCardLanguageVersion: '',
-			inviterIsGuestOrAnonymous: false,
-			basketedLoans: []
+			shareCardLanguageVersion: ''
 		};
 	},
 	computed: {
 		shareTitle() {
-			if (this.shareCardLanguageVersion === 'b') {
```

Comment:
Can the `shareCardLanguageVersion` be removed from this component now? I see the exp itself is being used in the thanks page, but these appear to be the only usages in this component.

Reply:
`this.shareCardLanguage` is still being used in this component to track the experiment. It can probably be removed? We might be able to remove it from this page and also from the thanks page, but check with the MARS team first

Reply:
Ah gotcha, it's an exp from a different team, thanks for the info.